src/client.js

```diff
@@ -8,8 +8,6 @@ export default class Client extends Port {
 
 		super(window.parent, '*', options);
 
-		this.lastHeight = 0;
```

Comment:
Leftover artifact from the initial move to iframe-resizer -- thanks @omsmith for noticing it.